test/i-frame.spec.ts

```diff
@@ -0,0 +1,44 @@
+import { browser } from 'protractor';
+import { IFramePage } from '../src/page';
+
+describe('Given a SDET learning protractor', () => {
+  const iFrame: IFramePage = new IFramePage();
+  describe('and then open Iframe page', () => {
+    beforeEach(async () => {
+      await browser.waitForAngularEnabled(false);
+      await browser.driver.get('https://demoqa.com/frames');
+    });
+
+    it('and then should have as title frames', async () => {
```

Comment:
```suggestion
    it('Then should have as title frames', async () => {
```
(Continuing with Gerkhin syntax)